docs/developer-docs/latest/getting-started/introduction.md

```diff
@@ -6,38 +6,38 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/getting-started/intro
+Strapi is a community-oriented project with an emphasis on transparency. The Strapi team has at heart to share their vision and build the future of Strapi with the Strapi community. This is why the [roadmap](https://portal.productboard.com/strapi) is open: as all insights are very important and help steer the project in the right direction, any community member is most welcome to share ideas and opinions there.
 
-Community members also take great part in providing the whole community a plethora of resources about Strapi. You can find [tutorials](https://strapi.io/tutorials/) on the Strapi website, where you can also create your own. Also, as an open-source project, the technical documentation of Strapi is open to contributions (see [Open-source & Contribution](#open-source-contribution)).
+Community members also take great part in providing the a plethora of resources about Strapi. You can find [tutorials](https://strapi.io/tutorials/) on the Strapi website, where you can even create your own. Also, as an open-source project, the technical documentation of Strapi is open to contributions (see [Open-source & Contribution](#open-source-contribution)).
 
 ::: strapi Want to join the community?
-You can join [GitHub](https://github.com/strapi/strapi), the [Forum](https://forum.strapi.io/), and the [Discord](https://discord.strapi.io) to share your ideas and opinions with other community members and members of the Strapi team. If you're looking for news and updates about Strapi, [Twitter](https://twitter.com/strapijs) and the [blog](https://strapi.io/blog) are pretty good places to start!
+You can join [GitHub](https://github.com/strapi/strapi), the [Forum](https://forum.strapi.io/), and the [Discord](https://discord.strapi.io) to share your ideas and opinions with other community members and members of the Strapi team. If you are looking for news and updates about Strapi, [Twitter](https://twitter.com/strapijs) and the [blog](https://strapi.io/blog) are pretty good places to start!
 :::
 
 ## Support
 
-Strapi is offered as free and open-source for users who wish to self-host the software. When having an issue or a question, the [forum](https://forum.strapi.io) is great first place to reach out for help. Both the Strapi community and core developers often check this platform and answer posts.
+Strapi is open-source and free and is therefore ideal for self-hosting, if a user wishes to do so. When having an issue or a question, the [forum](https://forum.strapi.io) is a great place to reach out for help. Both the Strapi community and core developers often check this platform and answer posts. [Stackoverflow](https://www.stackoverflow.com/) is another good place for getting answers to problems, too.
```

Comment:
I actually like it that you decided to not mention stackoverflow...

Reply:
Words like “ideal” and “great” should be avoided in technical writing because the documentation is mostly there to state facts about Strapi as a company and product. We’re not doing any kind of marketing communication, that’s why we try to stick to a neutral tone of voice and choice of words. Could you try to rephrase that sentence again? Otherwise we can keep the original version.